test/e2e/qat/qatplugin_dpdk.go

```diff
@@ -165,6 +166,34 @@ func describeQatDpdkPlugin() {
 			ginkgo.It("does nothing", func() {})
 		})
 	})
+
+	ginkgo.Context("When QAT's autoreset is on [Functionality:auto-reset]", func() {
+		ginkgo.BeforeEach(func() {
+			ginkgo.By("creating a configMap before plugin gets deployed")
+			e2ekubectl.RunKubectlOrDie(f.Namespace.Name, "create", "configmap", "--from-literal", "qat.conf=AutoresetEnabled=on", "qat-config")
+		})
+
+		ginkgo.It("checks if an injected error gets solved", func(ctx context.Context) {
+			injectError(ctx, f)
+			if err := utils.WaitForNodesWithResource(ctx, f.ClientSet, resourceName, 300*time.Second, utils.WaitForPositiveResource); err != nil {
+				framework.Failf("unable to wait for nodes to have positive allocatable resource: %v", err)
+			}
+		})
+	})
+
+	ginkgo.Context("When QAT's autoreset is on [Functionality:auto-reset]", func() {
+		ginkgo.BeforeEach(func() {
+			ginkgo.By("creating a configMap before plugin gets deployed")
+			e2ekubectl.RunKubectlOrDie(f.Namespace.Name, "create", "configmap", "--from-literal", "qat.conf=AutoresetEnabled=on", "qat-config")
+		})
```

Comment:
as far as I understand it, this test should be with `off` settings, right?

Reply:
yeap, thanks!